todo-app/src/App.js

```diff
@@ -1,10 +1,49 @@
 import React, { Component } from 'react';
+import Task from './Task';
+import tasks from './task.json';
 
 class App extends Component {
```

Comment:
Not terribly important, but just so you know, you can do your exporting inline on a class:

    export default class App extends Component

I know you probably pulled some of these examples from a tutorial, but in other files (`TaskDate.js`) you're exporting inline and here you're not, so just choose one method and use it throughout.